src/Symfony/Bundle/WebProfilerBundle/Tests/DependencyInjection/ConfigurationTest.php

```diff
@@ -26,18 +26,42 @@ public function testConfigTree($options, $results)
         $configuration = new Configuration();
         $config = $processor->processConfiguration($configuration, array($options));
 
+        // 'position' option is added by Symfony because it has a default value (remove it because it is deprecated)
+        unset($config['position']);
```

Comment:
can be removed? we can/should still test the default value.. no?

Reply:
right